docs/templating/templates/create_custom_view_matcher.md

```diff
@@ -27,11 +28,24 @@ This matcher identifies Content items that have the provided owner or owners.
 The matcher checks whether the owner of the current content (by its ContentInfo or Location)
 matches any of the values passed in configuration (line 44).
 
+## Matcher service
+
+You can configure your matcher as a service to have the opportunity to give him a matcher identifier easier to use than its fully qualified class name.
```

Comment:
but tbh, as mentioned earlier, I'd like to deprecate class-based matchers and not mention them.

Reply:
OK, I'll rephrase to get rid of FQCN.

Reply:
FQCN removed in 979e0f21adae56518d5e66f8a612c15eeeabfbbf